test/node/scenarios.json

```diff
@@ -21,7 +22,8 @@
         "features": {
             "node": {
                 "version": "lts",
-                "additionalVersions": "v17.9.1,v14.19.3"
+                "additionalVersions": "v17.9.1,v14.19.3",
+                "pnpmVersion": "6.16.0"
```

Comment:
Can we update the tests to verify the required version is installed?

Reply:
required version is verified in all the test cases for pnpmVersion